src/main/java/uk/gov/hmcts/ccd/security/JwtGrantedAuthoritiesConverter.java

```diff
@@ -33,6 +36,8 @@ public JwtGrantedAuthoritiesConverter(IdamRepository idamRepository) {
         List<GrantedAuthority> authorities = new ArrayList<>();
         if (jwt.containsClaim(TOKEN_NAME) && jwt.getClaim(TOKEN_NAME).equals(ACCESS_TOKEN)) {
             UserInfo userInfo = idamRepository.getUserInfo(jwt.getTokenValue());
+            log.info("JwtGrantedAuthoritiesConverter retrieved user info from idamRepository. User Id={}. Roles={}.",
```

Comment:
we are already logging in idamRepository, it is not needed again here... it will log the same info multiple times